smug.go

```diff
@@ -57,6 +59,70 @@ func (smug Smug) switchOrAttach(sessionName string, attach bool, insideTmuxSessi
 	return nil
 }
 
+//Exists verifies if file exists
```

Comment:
Commands like this can be removed because they don't any information. It's pretty clear from the method name what it does. Also, IsFileExists would be a better name